creator-node/src/config.js

```diff
@@ -826,7 +826,7 @@ const config = convict({
     format: String,
     env: 'reconfigSPIdBlacklistString',
     default:
-      '1,4,7,12,13,14,15,16,19,28,33,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,57,58,59,60,61,63,64,65'
+      '33,37,39,40,41,42,43,52,56,58,59,60,61,64,65'
```

Comment:
i thought we wanted to keep our nodes that are still running behind in blacklist? spIDs 1 & 4

Reply:
oh yes thank you. adding now